benchmark/single-source/JSON.swift

```diff
@@ -0,0 +1,88 @@
+//===--- JSON.swift ------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2018 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import TestsUtils
+import Foundation
+
+public let JSON = [
+  BenchmarkInfo(name: "JSONPerfEncode", runFunction: run_JSONPerfEncode, tags: [.validation, .bridging], setUpFunction: setup_coder),
+  BenchmarkInfo(name: "JSONPerfDecode", runFunction: run_JSONPerfDecode, tags: [.validation, .bridging], setUpFunction: setup_coder),
+]
+
+
+struct Tove : Codable {
```

Comment:
Style nit: the `:` should not have a space to the left
```suggestion
struct Tove: Codable {
```

Reply:
Same goes for the rest of the file